### PR TITLE
Minor refactor of resolve_remote_source plugin

### PR DESCRIPTION
**Pull request by chmeliik**
# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [n/a] JSON/YAML configuration changes are updated in the relevant schema
- [n/a] Changes to metadata also update the documentation for the metadata
- [n/a] Pull request has a link to an osbs-docs PR for user documentation updates
- [n/a] New feature can be disabled from a configuration file

### CODE REVIEW

**Comment by chmeliik** [comment]
PTAL, there are no basically no changes in behavior here, just refactoring to make the next change (merging with `download_remote_sources`) easier.

**Review by mkosiarc — APPROVED** [review]

File: atomic_reactor/plugins/pre_resolve_remote_source.py [hunk 1]
```diff
@@ -1,24 +1,58 @@
 """
-Copyright (c) 2019 Red Hat, Inc
+Copyright (c) 2019, 2021 Red Hat, Inc
```
Comment (hjmodi) [inline atomic_reactor/plugins/pre_resolve_remote_source.py:2]: nitpick: we should add 2022 here

**Review by hjmodi — APPROVED** [review]

File: atomic_reactor/plugins/pre_resolve_remote_source.py (hunk 1 above)
Comment (chmeliik) [reply atomic_reactor/plugins/pre_resolve_remote_source.py]: Good catch, fixed

**Comment by chmeliik** [comment]
Rebased on master and fixed dates in licenses. Will merge if CI succeeds.

**Pull request merged**